.jenkins/Jenkinsfile

```diff
@@ -16,524 +16,486 @@
+           testJob(".build/docker/run-tests.sh", "test-burn", 8)
+         }
+       }
+       stage('cdc') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test-cdc", 8)
+         }
+       }
+       stage('compression') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "test-compression", 8)
+         }
+       }
+       stage('cqlsh') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "cqlsh-test", 1)
+         }
+       }
+     }
+    }
+    // stage('Distributed Test') {
+    //  parallel {
+    //    stage('jvm-dtest') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "jvm-dtest", 8)
+    //      }
+    //    }
+    //    stage('jvm-dtest-upgrade') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "jvm-dtest-upgrade", 8)
+    //      }
+    //    }
+    //    stage('dtest') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest", 64)
+    //      }
+    //    }
+    //    stage('dtest-large') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-large", 8)
+    //      }
+    //    }
+    //    stage('dtest-novnode') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-novnode", 64)
+    //      }
+    //    }
+    //    stage('dtest-offheap') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-offheap", 64)
+    //      }
+    //    }
+    //    stage('dtest-large-novnode') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-large-novnode", 8)
+    //      }
+    //    }
+    //    stage('dtest-upgrade') {
+    //      steps {
+    //        testJob(".build/docker/run-tests.sh", "dtest-upgrade", 64)
+    //      }
+    //    }
+    //  }
+    // }
+    stage('Summary') {
+      steps {
+       // FIXME is this needed if each job is doing it ??
+       //junit testResults: '**/build/test/**/TEST*.xml,**/cqlshlib.xml,**/nosetests.xml', testDataPublishers: [[$class: 'StabilityTestDataPublisher']]
+
+       //sendNotifications()
+       
+       // FIXME how to collect and build one test report file ?? 
+       //generateUnifiedTestReport()
+       
+        collectConsoleLog()
+        copyToNightlies('console.log.xz,TESTS-TestSuites.xml.xz')
+      }
+    }
+  }
+}
+
+////
+//// scripting support ////
+////
+
+
+/**
+ * Return the default JDK defined by build.xml
+ **/
+def javaVersionDefault() {
+  sh (returnStdout: true, script: 'grep \'property\\s*name=\"java.default\"\' build.xml | sed -ne \'s/.*value=\"\\([^\"]*\\)\".*/\\1/p\'').trim()
+}
+
+/**
+ * Return the supported JDKs defined by build.xml
+ **/
+def javaVersionsSupported() {
+  sh (returnStdout: true, script: 'grep \'property\\s*name=\"java.supported\"\' build.xml | sed -ne \'s/.*value=\"\\([^\"]*\\)\".*/\\1/p\'').trim()
+}
+
+/**
+ * Is this a post-commit build (or a pre-commit build)
+ **/
+def isPostCommit() {
```

Comment:
Could use a bit more description for why this works. Because I don't get it. :)

Reply:
dev branches (i.e. pre-commit) won't be found in the origin "apache" repo.

Reply:
> dev branches (i.e. pre-commit) won't be found in the origin "apache" repo.

Comment that in the script?